src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -293,14 +298,26 @@ export default function formReducer(state = initialState, action) {
       let newSchema = action.schema;
       let newData = state.data;
       let { facilities } = action;
-      const { typeOfCareId } = action;
-      const { address } = action;
-      const { cernerSiteIds } = action;
+      const {
+        typeOfCareId,
+        featureRecentLocationsFilter,
+        recentLocations,
+        address,
+        cernerSiteIds,
+      } = action;
+      let recentFacilities = recentLocations;
       const hasResidentialCoordinates =
         !!action.address?.latitude && !!action.address?.longitude;
-      const sortMethod = hasResidentialCoordinates
-        ? FACILITY_SORT_METHODS.distanceFromResidential
-        : FACILITY_SORT_METHODS.alphabetical;
+      const handleSortMethod = () => {
```

Comment:
nit: Hmm code here seems a little unnecessary, how about

```suggestion
      let sortMethod = FACILITY_SORT_METHODS.alphabetical;
      if (featureRecentLocationsFilter && recentLocations?.length) {
        sortMethod = FACILITY_SORT_METHODS.recentLocations;
      }
      if (hasResidentialCoordinates) {
        sortMethod = FACILITY_SORT_METHODS.distanceFromResidential;
      }
```